tests/smoke_tests/parcel/jensen_and_nugent_2017/test_fig_5.py

```diff
@@ -0,0 +1,92 @@
+# pylint: disable=missing-module-docstring,missing-class-docstring,missing-function-docstring
+
+from pathlib import Path
+
+import numpy as np
+import pytest
+from scipy import signal
+
+from PySDM_examples.utils import notebook_vars
+from PySDM_examples import Jensen_and_Nugent_2017
+from PySDM.physics.constants import PER_CENT
+
+from PySDM.physics import si
+
+PLOT = False
+N_SD = Jensen_and_Nugent_2017.simulation.N_SD_NON_GCCN + np.count_nonzero(
+    Jensen_and_Nugent_2017.table_3.NA
+)
+
+
+def find_cloud_base_index(products):
```

Comment:
this function is copy-pasted from another file - please replace with an import

Reply:
done